js_modules/dagster-ui/packages/ui-core/src/app/observeEnabled.oss.tsx

```diff
@@ -0,0 +1,11 @@
+import {FeatureFlag} from 'shared/app/FeatureFlags.oss';
+
+import {featureEnabled} from './Flags';
+
+/**
+ * Function to check if a specific feature flag is enabled.
+ * Falls back to default values if the flag is unset.
+ */
+export const observeEnabled = (): boolean => {
+  return featureEnabled(FeatureFlag.flagUseNewObserveUIs);
```

Comment:
I'm thinking this should always return false in OSS.

When running the app in Cloud, any code that imports `observeEnabled` will get it from the `.cloud` version and correctly check the flag. But in the OSS case, I think we wouldn't want Observe to be enabled if the user happens to (for some reason) have the flag enabled in localStorage.